src/enigma2/Channels.cpp

```diff
@@ -153,8 +153,7 @@ bool Channels::LoadChannels(const std::string groupServiceReference, const std::
 {
   Logger::Log(LEVEL_INFO, "%s loading channel group: '%s'", __FUNCTION__, groupName.c_str());
 
-  std::string strTmp;
-  strTmp = StringUtils::Format("%sweb/getservices?sRef=%s", Settings::GetInstance().GetConnectionURL().c_str(), WebUtils::URLEncodeInline(groupServiceReference).c_str());
+  std::string strTmp = StringUtils::Format("%sweb/getservices?sRef=%s", Settings::GetInstance().GetConnectionURL().c_str(), WebUtils::URLEncodeInline(groupServiceReference).c_str());
```

Comment:
`const std::string`?

Reply:
I'll go through the go base and clean this up in general. Been meaning to do it for a while. Will include in next PR